core/src/main/AndroidManifest.xml

```diff
@@ -10,16 +10,8 @@
     <!-- SERVICE -->
```

Comment:
We _could_ take these out, it's not like one would think a `<service>` is anything but a Service :P